docs/content/how-to/node-tuning.md

```diff
@@ -95,4 +96,126 @@ If you would like to set some node-level tuning on the nodes in your hosted clus
     Example output:
     ```
     vm.dirty_ratio = 55
+    ```
+
+## Applying tuning which requires kernel boot parameters 
+You can also use the Node Tuning Operator for more complex tuning which requires setting kernel boot parameters. 
+As an example, the following steps can be followed to create a NodePool with huge pages reserved.
+
+1. Create the following ConfigMap which contains a Tuned object manifest for creating 10 hugepages of size 2M.
+
+    Save this ConfigMap manifest in a file called `tuned-hugepages.yaml`:
+    ```
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: tuned-hugepages
+      namespace: clusters
+    data:
+      tuned: |
+        apiVersion: tuned.openshift.io/v1
+        kind: Tuned
+        metadata:
+          name: hugepages
+          namespace: openshift-cluster-node-tuning-operator
+        spec:
+          profile:
+          - data: |
+              [main]
+              summary=Boot time configuration for hugepages
+              include=openshift-node
+              [bootloader]
+              cmdline_openshift_node_hugepages=hugepagesz=2M hugepages=50
+            name: openshift-node-hugepages
+          recommend:
+          - priority: 20
+            profile: openshift-node-hugepages
+    ```
+    > **_NOTE:_**  The `.spec.recommend.match` field is intentionally left blank. In this case this Tuned will be applied to all Nodes in the NodePool where this ConfigMap is referenced. It is advised to group Nodes with the same hardware configuration into the same NodePool. Not following this practice might result in TuneD operands calculating conflicting kernel parameters for two or more nodes sharing the same NodePool.
```

Comment:
This is IMPORTANT, but let's leave it just a NOTE.  This bogged me down as the first "user" of the new code.  An aside, yesterday I was testing with the `.spec.recommend.match` field targetting a single node in the node pool.  It is probably the reason I was [getting this](https://github.com/openshift/cluster-node-tuning-operator/pull/456#pullrequestreview-1097466483) and something that still needs to be addressed on the NTO side.

Reply:
There should be no possible choice for the user to target particular nodes within a NodePool.

Reply:
This is behaviour is based off of how NTO works in standalone OCP. There are many cases where a TuneD profile is making in-place changes to node tunables and no MachineConfig is needed. For example, if a user wants to set some sysctl values on one node with particular labels and assign some Pods only to that Node by label. 

If we do decide to remove this feature in HyperShift, we can do that, but it would be a change to the NTO code.

Reply:
Also note that the original issue Jiri hit here was fixed. If a user does use Node label based matching, no MachineConfig will be generated based on that Profile